### PR TITLE
Change dashboard to short number notation

### PR DESCRIPTION
**Pull request by noellabo**
The number on the dashboard overflows on the smartphone.

It is especially useful if the number of digits in interactions this week is large and you can switch to a short notation.

### CODE REVIEW

**Comment by Gargron** [comment]
I think you can skip the setting part. Make it always use short notation with full notation in `title`.

**Comment by noellabo** [comment]
I agree. Let's make a simple change.

File: app/helpers/admin/dashboard_helper.rb [hunk 1]
```diff
@@ -7,4 +7,8 @@ def feature_hint(feature, enabled)
 
     safe_join([feature, content_tag(:span, indicator, class: class_names)])
   end
+
+  def short_number_format(number)
+    number_to_human number, units: { unit: '', thousand: 'K', million: 'M', billion: 'B' }
```
Comment (Gargron) [inline app/helpers/admin/dashboard_helper.rb]: Why is this necessary? Followers count on profile pages already uses short number format

Comment (noellabo) [reply app/helpers/admin/dashboard_helper.rb]: It existed for switching between short and full notation. Match the way you write your profile page.

**Review by Gargron — APPROVED** [review]
